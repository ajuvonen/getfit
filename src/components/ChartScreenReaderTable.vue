<script lang="ts" setup>
defineProps<{
  id: string,
  title: string,
  labels: string[],
  data: number[],
}>();
</script>
<template>
  <table class="d-sr-only" :id="id">
    <caption>{{ title }}</caption>
    <thead>
      <tr>
        <th v-for="label in labels" :key="label">{{ label }}</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td v-for="count in data" :key="count">{{ count }}</td>
      </tr>
    </tbody>
  </table>
</template>
