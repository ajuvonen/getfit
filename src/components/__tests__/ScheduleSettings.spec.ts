import {nextTick} from 'vue';
import {mount} from '@vue/test-utils';
import {describe, it, expect, beforeEach} from 'vitest';
import ScheduleSettings from '@/components/ScheduleSettings.vue';
import {useScheduleStore} from '@/stores/schedule';
import {useAppStateStore} from '@/stores/appState';
import {VSwitch} from 'vuetify/components/VSwitch';

describe('ScheduleSettings', () => {
  const scheduleStore = useScheduleStore();
  const appStateStore = useAppStateStore();

  beforeEach(() => {
    appStateStore.$reset();
    scheduleStore.$reset();
  });

  it('mounts', () => {
    const wrapper = mount(ScheduleSettings);
    expect(wrapper.html()).toMatchSnapshot();
  });

  // it('hides actions when schedule is locked', async () => {
  //   const wrapper = mount(ScheduleSettings);
  //   expect(wrapper.findByTestId('schedule-settings-add-week-button').exists()).toBe(true);

  //   wrapper.findComponent(VSwitch).vm.$emit('update:modelValue');
  //   expect(scheduleStore.toggleLockSchedule).toHaveBeenCalled();
  //   scheduleStore.schedule.lockSchedule = true;
  //   await nextTick();
  //   expect(wrapper.findByTestId('schedule-settings-add-week-button').exists()).toBe(false);
  // });
});
