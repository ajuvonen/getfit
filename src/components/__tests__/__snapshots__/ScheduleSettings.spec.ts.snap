// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`EditTrainingDialog > mounts 1`] = `
"<div data-v-76426b41=\\"\\" class=\\"v-card v-theme--light v-card--density-default v-card--variant-elevated mx-auto mt-10 mb-10\\">
  <!---->
  <div class=\\"v-card__loader\\">
    <div class=\\"v-progress-linear v-theme--light v-locale--is-ltr\\" style=\\"top: 0px; height: 0px; --v-progress-linear-height: 2px; left: 50%; transform: translateX(-50%);\\" role=\\"progressbar\\" aria-hidden=\\"true\\" aria-valuemin=\\"0\\" aria-valuemax=\\"100\\">
      <!---->
      <div class=\\"v-progress-linear__background\\" style=\\"width: 100%;\\"></div>
      <transition-stub name=\\"fade-transition\\" appear=\\"false\\" persisted=\\"false\\" css=\\"true\\">
        <div class=\\"v-progress-linear__indeterminate\\">
          <div class=\\"v-progress-linear__indeterminate long\\"></div>
          <div class=\\"v-progress-linear__indeterminate short\\"></div>
        </div>
      </transition-stub>
      <!---->
    </div>
  </div>
  <!---->
  <!---->
  <div data-v-76426b41=\\"\\" class=\\"v-card-text\\">
    <div data-v-76426b41=\\"\\" class=\\"v-expansion-panels v-theme--light v-expansion-panels--variant-default\\">
      <div data-v-76426b41=\\"\\" class=\\"v-expansion-panel v-expansion-panel--active\\">
        <div class=\\"v-expansion-panel__shadow elevation-0\\"></div>
        <!---->
        <!----><button data-v-76426b41=\\"\\" class=\\"v-expansion-panel-title v-expansion-panel-title--active\\" type=\\"button\\" aria-expanded=\\"true\\"><span class=\\"v-expansion-panel-title__overlay\\"></span>
          <h2 data-v-76426b41=\\"\\" class=\\"text-h5\\">Schedule Settings</h2><span class=\\"v-expansion-panel-title__icon\\"><i class=\\"mdi-chevron-up mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i></span>
        </button>
        <transition-stub data-v-76426b41=\\"\\" name=\\"expand-transition\\" appear=\\"false\\" persisted=\\"false\\" css=\\"true\\">
          <div class=\\"v-expansion-panel-text\\">
            <div class=\\"v-expansion-panel-text__wrapper\\">
              <div data-v-76426b41=\\"\\" class=\\"v-input v-input--horizontal v-input--density-default v-text-field v-text-field--flush-details\\">
                <!---->
                <div class=\\"v-input__control\\">
                  <div class=\\"v-field v-field--variant-underlined v-theme--light v-locale--is-ltr\\" role=\\"textbox\\">
                    <div class=\\"v-field__overlay\\"></div>
                    <div class=\\"v-field__loader\\">
                      <div class=\\"v-progress-linear v-theme--light v-locale--is-ltr\\" style=\\"top: 0px; height: 0px; --v-progress-linear-height: 2px; left: 50%; transform: translateX(-50%);\\" role=\\"progressbar\\" aria-hidden=\\"true\\" aria-valuemin=\\"0\\" aria-valuemax=\\"100\\">
                        <!---->
                        <div class=\\"v-progress-linear__background\\" style=\\"width: 100%;\\"></div>
                        <transition-stub name=\\"fade-transition\\" appear=\\"false\\" persisted=\\"false\\" css=\\"true\\">
                          <div class=\\"v-progress-linear__indeterminate\\">
                            <div class=\\"v-progress-linear__indeterminate long\\"></div>
                            <div class=\\"v-progress-linear__indeterminate short\\"></div>
                          </div>
                        </transition-stub>
                        <!---->
                      </div>
                    </div>
                    <!---->
                    <div class=\\"v-field__field\\" data-no-activator=\\"\\">
                      <!----><label class=\\"v-label v-field-label\\" for=\\"schedule-settings-program-name\\">
                        <!---->Program Name
                      </label>
                      <!----><input size=\\"1\\" type=\\"text\\" id=\\"schedule-settings-program-name\\" aria-describedby=\\"schedule-settings-program-name-messages\\" maxlength=\\"30\\" class=\\"v-field__input\\">
                      <!---->
                    </div>
                    <!---->
                    <!---->
                    <div class=\\"v-field__outline\\">
                      <!----><label class=\\"v-label v-field-label v-field-label--floating\\" aria-hidden=\\"true\\" for=\\"schedule-settings-program-name\\">
                        <!---->Program Name
                      </label>
                    </div>
                  </div>
                </div>
                <!---->
                <div class=\\"v-input__details\\">
                  <transition-group-stub name=\\"slide-y-transition\\" tag=\\"div\\" appear=\\"false\\" persisted=\\"false\\" css=\\"true\\" class=\\"v-messages\\" role=\\"alert\\" aria-live=\\"polite\\" id=\\"schedule-settings-program-name-messages\\">
                    <!---->
                  </transition-group-stub>
                  <!----><span></span>
                  <transition-stub name=\\"slide-y-transition\\" appear=\\"false\\" persisted=\\"false\\" css=\\"true\\">
                    <div class=\\"v-counter\\" style=\\"display: none;\\">0 / 30</div>
                  </transition-stub>
                </div>
              </div><label data-v-76426b41=\\"\\" class=\\"v-label\\" for=\\"schedule-settings-start-of-week\\">
                <!---->Start of Week
              </label>
              <div data-v-76426b41=\\"\\" class=\\"v-input v-input--horizontal v-input--density-default v-input--dirty v-radio-group\\">
                <!---->
                <div class=\\"v-input__control\\">
                  <!---->
                  <div class=\\"v-selection-control-group v-selection-control-group--inline\\" role=\\"radiogroup\\" aria-describedby=\\"schedule-settings-start-of-week-messages\\">
                    <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--inline v-selection-control--density-default v-radio\\" data-test-id=\\"schedule-settings-start-of-week-monday\\">
                      <div class=\\"v-selection-control__wrapper\\">
                        <!---->
                        <div class=\\"v-selection-control__input\\"><i class=\\"mdi-radiobox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"input-6\\" aria-disabled=\\"false\\" type=\\"radio\\" name=\\"schedule-settings-start-of-week\\" value=\\"false\\">
                          <!---->
                        </div>
                      </div><label class=\\"v-label v-label--clickable\\" for=\\"input-6\\">
                        <!---->Monday
                      </label>
                    </div>
                    <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--inline v-selection-control--density-default v-radio\\" data-test-id=\\"schedule-settings-start-of-week-sunday\\">
                      <div class=\\"v-selection-control__wrapper\\">
                        <!---->
                        <div class=\\"v-selection-control__input\\"><i class=\\"mdi-radiobox-blank mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"input-7\\" aria-disabled=\\"false\\" type=\\"radio\\" name=\\"schedule-settings-start-of-week\\" value=\\"true\\">
                          <!---->
                        </div>
                      </div><label class=\\"v-label v-label--clickable\\" for=\\"input-7\\">
                        <!---->Sunday
                      </label>
                    </div>
                  </div>
                </div>
                <!---->
                <div class=\\"v-input__details\\">
                  <transition-group-stub name=\\"slide-y-transition\\" tag=\\"div\\" appear=\\"false\\" persisted=\\"false\\" css=\\"true\\" class=\\"v-messages\\" role=\\"alert\\" aria-live=\\"polite\\" id=\\"schedule-settings-start-of-week-messages\\">
                    <!---->
                  </transition-group-stub>
                  <!---->
                </div>
              </div><label data-v-76426b41=\\"\\" class=\\"v-label\\" for=\\"schedule-settings-unit-of-time\\">
                <!---->Unit of Time
              </label>
              <div data-v-76426b41=\\"\\" class=\\"v-input v-input--horizontal v-input--density-default v-input--dirty v-radio-group\\">
                <!---->
                <div class=\\"v-input__control\\">
                  <!---->
                  <div class=\\"v-selection-control-group v-selection-control-group--inline\\" role=\\"radiogroup\\" aria-describedby=\\"schedule-settings-unit-of-time-messages\\">
                    <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--inline v-selection-control--density-default v-radio\\" data-test-id=\\"schedule-settings-unit-of-time-h\\">
                      <div class=\\"v-selection-control__wrapper\\">
                        <!---->
                        <div class=\\"v-selection-control__input\\"><i class=\\"mdi-radiobox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"input-11\\" aria-disabled=\\"false\\" type=\\"radio\\" name=\\"schedule-settings-unit-of-time\\" value=\\"h\\">
                          <!---->
                        </div>
                      </div><label class=\\"v-label v-label--clickable\\" for=\\"input-11\\">
                        <!---->Hours
                      </label>
                    </div>
                    <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--inline v-selection-control--density-default v-radio\\" data-test-id=\\"schedule-settings-unit-of-time-m\\">
                      <div class=\\"v-selection-control__wrapper\\">
                        <!---->
                        <div class=\\"v-selection-control__input\\"><i class=\\"mdi-radiobox-blank mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"input-12\\" aria-disabled=\\"false\\" type=\\"radio\\" name=\\"schedule-settings-unit-of-time\\" value=\\"m\\">
                          <!---->
                        </div>
                      </div><label class=\\"v-label v-label--clickable\\" for=\\"input-12\\">
                        <!---->Minutes
                      </label>
                    </div>
                  </div>
                </div>
                <!---->
                <div class=\\"v-input__details\\">
                  <transition-group-stub name=\\"slide-y-transition\\" tag=\\"div\\" appear=\\"false\\" persisted=\\"false\\" css=\\"true\\" class=\\"v-messages\\" role=\\"alert\\" aria-live=\\"polite\\" id=\\"schedule-settings-unit-of-time-messages\\">
                    <!---->
                  </transition-group-stub>
                  <!---->
                </div>
              </div><label data-v-76426b41=\\"\\" class=\\"v-label\\">
                <!---->Available Activities
              </label>
              <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default v-checkbox-btn\\" data-test-id=\\"schedule-settings-toggle-all-activities\\">
                <div class=\\"v-selection-control__wrapper text-secondary\\">
                  <!---->
                  <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"input-13\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" hide-details=\\"auto\\" value=\\"true\\">
                    <!---->
                  </div>
                </div><label class=\\"v-label v-label--clickable\\" for=\\"input-13\\">
                  <!---->Toggle All
                </label>
              </div>
              <div data-v-76426b41=\\"\\" role=\\"group\\">
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-badminton\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"badminton\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-badminton\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-badminton mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Badminton
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-basketball\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"basketball\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-basketball\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-basketball mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Basketball
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-maintenance\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"maintenance\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-maintenance\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-yoga mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Body Maintenance
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-boxing\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"boxing\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-boxing\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-boxing-glove mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Boxing
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-downhill\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"downhill\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-downhill\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-ski mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Downhill
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-gym\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"gym\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-gym\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-dumbbell mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Gym
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-hockey\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"hockey\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-hockey\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-hockey-sticks mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Hockey
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-martialarts\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"martialarts\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-martialarts\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-karate mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Martial Arts
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-meditation\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"meditation\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-meditation\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-meditation mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Meditation
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-running\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"running\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-running\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-run mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Running
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-skiing\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"skiing\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-skiing\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-ski-cross-country mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Skiing
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-soccer\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"soccer\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-soccer\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-soccer mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Soccer
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-sprint\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"sprint\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-sprint\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-run-fast mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Sprint
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-swimming\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"swimming\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-swimming\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-swim mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Swimming
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-tennis\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"tennis\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-tennis\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-tennis mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Tennis
                  </label>
                </div>
                <div data-v-76426b41=\\"\\" class=\\"schedule-settings__icon-container\\">
                  <div data-v-76426b41=\\"\\" class=\\"v-selection-control v-selection-control--dirty v-selection-control--density-default flex-grow-0 v-checkbox-btn flex-grow-0\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <!---->
                      <div class=\\"v-selection-control__input\\"><i class=\\"mdi-checkbox-marked mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i><input id=\\"schedule-settings-activity-walking\\" aria-disabled=\\"false\\" type=\\"checkbox\\" indeterminateicon=\\"$checkboxIndeterminate\\" value=\\"walking\\">
                        <!---->
                      </div>
                    </div>
                    <!---->
                  </div><label data-v-76426b41=\\"\\" class=\\"v-label flex-grow-1\\" for=\\"schedule-settings-activity-walking\\">
                    <!----><i data-v-76426b41=\\"\\" class=\\"mdi-walk mdi v-icon notranslate v-theme--light v-icon--size-large\\" aria-hidden=\\"true\\"></i> Walking
                  </label>
                </div>
              </div><label data-v-76426b41=\\"\\" class=\\"v-label\\" for=\\"schedule-settings-lock-schedule-button\\">
                <!---->Lock Schedule
              </label>
              <div data-v-76426b41=\\"\\" class=\\"v-input v-input--horizontal v-input--density-default v-input--dirty v-switch\\">
                <!---->
                <div class=\\"v-input__control\\">
                  <div class=\\"v-selection-control v-selection-control--density-default\\">
                    <div class=\\"v-selection-control__wrapper\\">
                      <div class=\\"v-switch__track\\"></div>
                      <div class=\\"v-selection-control__input\\">
                        <!----><input id=\\"schedule-settings-lock-schedule-button\\" aria-disabled=\\"false\\" type=\\"checkbox\\" aria-describedby=\\"schedule-settings-lock-schedule-button-messages\\" value=\\"true\\">
                        <div class=\\"v-switch__thumb\\">
                          <!---->
                        </div>
                      </div>
                    </div>
                    <!---->
                  </div>
                </div>
                <!---->
                <!---->
              </div>
            </div>
          </div>
        </transition-stub>
      </div>
    </div>
  </div>
  <div data-v-76426b41=\\"\\" class=\\"v-card-actions\\"><button data-v-76426b41=\\"\\" type=\\"button\\" class=\\"v-btn v-theme--light v-btn--density-default v-btn--size-default v-btn--variant-text\\" data-test-id=\\"schedule-settings-add-week-button\\"><span class=\\"v-btn__overlay\\"></span><span class=\\"v-btn__underlay\\"></span><span class=\\"v-btn__prepend\\"><i class=\\"mdi-plus mdi v-icon notranslate v-theme--light v-icon--size-default\\" aria-hidden=\\"true\\"></i></span><span class=\\"v-btn__content\\" data-no-activator=\\"\\">Add Week</span>
      <!---->
      <!---->
    </button></div>
  <!---->
  <!----><span class=\\"v-card__underlay\\"></span>
</div>"
`;
